### ISSUE TITLE
Requête fonctionnalité

### ISSUE THREAD

**Issue by thowtue** [Hyrules/WinHue3#141]
Bonjour !
J'ai quelque chose à proposer, qui pourrait plaire à certaines personnes y comprit moi,
J'utilise une autre application (huedynamic) qui contrôle les lampes Hue mais je l'utilise uniquement pour une seule fonctionnalité c'est celle qui permet de faire quelque chose de similaire au système ambilight de Philips, il y a un programme en arrière plan qui synchronise en temps réel les ampoules Hue avec ce qui est affiché sur l'écran, et ça crée une ambiance...

Je pense que c'est une fonctionnalité qui demande pas mal de temps à dev, et a intégrer des options pour régler indépendamment les ampoules selon les zones de l'écran à capturer etc, etc.
Voilà voilà, si besoin de plus d'infos, où pour des tests je suis là !
Merci !

**Comment by Hyrules** [comment]
Je vais voir ce que je peux faire. J'ai une idée derrière le fonctionnement. Il faut faire une capture d'écran et analyser les pixels et changer la couleur des ampoules en conséquence. Maintenant comment le faire en C# c'est une autre chose.

**Comment by thowtue** [comment]
Oui, c'est sûr que c'est pas facile, et en effet je ne sais pas si c'est faisable en C#. 
Petite indication, l'application est aussi en C# et se trouve sur le WinStore, mais le programme de capture (même si je sais pas en quel langage il est codé), c'est une application séparée à installer via un .msi et qui tourne en fond donc.

**Comment by Hyrules** [comment]
Faisable oui c'est très faisable. La question c'est comment. Il doit y a voir des biblios qui existent pour avoir l'image d'une video. Je vais voir ce que je suis capable de faire.

**Comment by thowtue** [comment]
D'accord, je peux regarder ou demander de mon côté aussi, juste que vu que je suis pas dev, même si j'ai quelques connaissances là dessus, je vais peut être avoir du mal à chercher ce que je voudrais et ce qu'il faudrait

**Comment by Hyrules** [comment]
En fait je crois que la biblio AForge.NET me permettrai de le faire. Je vais regarder plus en détail.

**Comment by thowtue** [comment]
D'accord ! Si besoin de quoi que ce soit, je suis dispo. :)

**Comment by ManuelGotzen** [comment]
**Ambilight** ist nicht einfach zu programmieren. Ich beschäftige mich schon seit vielen Jahren mit dieser Thematik. Zwar bin ich selber Programmierer,.wollte mich darin aber nicht  nicht vertiefen, ich bin ein Freund der Cloud.  Und das es bald eine solche Funktion geben würde, die auf einfache Weise **funktioniert** **und** **mit den gegebenen Geräten** konfigurierbar ist. 

Eine solche Lösung gibt es nicht. 
Und ich warte schon seit über 5 Jahren darauf. Gut... 

Gehen wir mal von folgender **System-Anforderung** aus: 

- **PhilipsHue Starter Set** (2xBulb und eine N14 Lampe) 
- **Google Chromecast** (egal welche Generation) 


**GitHub Ressourcen**: 
https://github.com/search?p=2&q=ambilight+philips+hue&ref=simplesearch&type=Repositories&utf8=%E2%9C%93

Hue Hilfe: 
http://bit.ly/check-hue

greetings, 
Manuel Gotzen

**Comment by Hyrules** [comment]
Sorry but I don't speak or understand German. If you want to suggest something please do so in English or French. Otherwise i'm not implementing any more feature in WinHue for now. Thanks.

**Comment by ManuelGotzen** [comment]
I'm **sorry**, **new** Smart**Phone** and the Google **Translator was not** configured yet. 

**Ambilight** is not easy to program. I have been dealing with this topic for many years. I'm a programmer, but I didn't want to delve into it, I'm a friend of the cloud (eclipse) And that there would soon be such a function **that works** in a **simple** way **and is configurable with the given devices**.

 There is no such solution. 
 And I've been waiting for it for over 5 years. _C'est bon_...

Let's assume the following **system requirement**: 

- **PhilipsHue** Starter Set (2xBulb und eine N14 Lampe) 

- **Google Chromecast** (egal welche Generation)


**GitHub Ressourcen**:
https://github.com/search?p=2&q=ambilight+philips+hue&ref=simplesearch&type=Repositories&utf8=%E2%9C%93

Hue helper:
http://bit.ly/check-hue

_greetings,_ 
Manuel Gotzen

**Comment by ManuelGotzen** [comment]
a whole projects and respositories. 
One of these #**AmbilightHue**. 

https://github.com/matthiaswelz/ambilighthue

https://github.com/romusters/hue/blob/master/README.mf

https://www.pyimagesearch.com/2014/05/26/opencv-python-k-means-color-clustering/ 

...

**Comment by ManuelGotzen** [comment]
> Sorry but I don't speak or understand German. If you want to suggest something please do so in English or French. Otherwise i'm not implementing any more feature in WinHue for now. Thanks.

I'm **sorry**, **new** Smart**Phone** and the Google **Translator was not** configured yet. 

**Ambilight** is not easy to program. I have been dealing with this topic for many years. I'm a programmer, but I didn't want to delve into it, I'm a friend of the cloud (eclipse) And that there would soon be such a function **that works** in a **simple** way **and is configurable with the given devices**.

 There is no such solution. 
 And I've been waiting for it for over 5 years. _C'est bon_...

Let's assume the following **system requirement**: 

- **PhilipsHue** Starter Set (2xBulb und eine N14 Lampe) 

- **Google Chromecast** (egal welche Generation)


**GitHub Ressourcen**:
https://github.com/search?p=2&q=ambilight+philips+hue&ref=simplesearch&type=Repositories&utf8=%E2%9C%93

**Hue helper**:
http://bit.ly/check-hue

_greetings,_ 
Manuel Gotzen
